Zappy_IA_Report/DTC/Code/process.py

```
@@ -0,0 +1,17 @@
+import sys
+from models import iaParser
+from parser import ia_parser
+from utils import exit_error
+
+def iaProcess(argv: list[str]) -> iaParser:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)